docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,53 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged to
+   into main.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Fill up the `CHANGELOG.md` file with latest changes.
+    - Run `generate-changelog.sh` to generate the change notes for your release.
+    - Use the `generate-changed-recipes` script to print the list of changes in
+      recipe versions. The script uses build manifests - lists of recipes
+      contained in the image along with their versions. You must provide your
+      manifest and the current release's manifest as arguments to the script.
+      The latter can be found in the latest release's section in the GitHub
+      release pages.
+    - Paste the script output into your release's section in `CHANGELOG.md`.
+      Additionally, you must explain why those changes were made and what they
+      introduce (unless the commit messages listed by `generate-changelog.sh`
+      already explain that).
+    - Describe any additional changes.
+4. Commit and push the changelog.
+5. Create and push tag that matches the newly bumped version.
+6. Publish the release on GitHub. Make sure to upload the manifest.
```

Comment:
@PLangowski, what about RC releases? Do we push as a draft release, or do we not push at all?

Reply:
Most of the process is the same, we just publish it as a pre-release https://github.com/zarhus/docs/compare/20e8d1eb800f1e034ce33d1d4bbe2617af90d8a1..9a81040be435abcac59cb8fd58b28a2cc01adf3d#diff-601babdf30d07c9e8dc632e099bf77c53afe5e0ec2dc70a2d37ea0cbfdbe3c5cR28